presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -8272,6 +8273,34 @@ public void testRightJoinWithEmptyProbeSide()
                 "WITH small_part AS (SELECT * FROM part WHERE name = 'a') SELECT lineitem.orderkey FROM small_part RIGHT JOIN lineitem ON  small_part.partkey = lineitem.partkey");
     }
 
+    @Test
+    public void testLastValueIgnoreNulls()
+    {
+        String query = "WITH T AS (" +
```

Comment:
You can probably write it similar as other tests using `assertQueryOrdered` and provide expected results, rather than just checking for `null` (look at `testValueWindowFunctions`).

Reply:
We should not need ordered. So testing for unordered